lib/intern.js

```diff
@@ -137,11 +137,17 @@ function shouldMerge(ent, options = null) {
 }
 
 
+function escapeIdentifier(s) {
+  return '"' + s + '"'
```

Comment:
let's make this safer - test if string and use JSON.stringify to generate
what happens if s contains a "?